integreat_cms/static/src/js/poi_box.ts

```diff
@@ -3,11 +3,11 @@ import { getCsrfToken } from "./utils/csrf-token";
 
```

Comment:
Nit: I think we said at one point, that the underscore should be used for Python/Django files and for JS/TS files, we just a dash instead. So, if I remember correctly, I think this should be `poi-box.ts`. I hope changing this won't give you a headache :see_no_evil: